### PR TITLE
add license wizard to scholar 3

### LINKED ISSUE #901: Add license wizard to work create/edit

### PR DESCRIPTION
**Pull request by lawhorkl**
Fixes #901 ;

Present short summary (50 characters or less)

Add license wizard from curate to scholar 3.

Changes proposed in this pull request:
* add license wizard button and modal action

### CODE REVIEW

File: spec/features/end_to_end_spec.rb [hunk 1]
```diff
@@ -49,6 +49,11 @@
       expect(page).to have_content('Add New Work')
     end
 
+    it 'can see the license wizard on the new work form' do
+      visit new_curation_concerns_work_path
+      expect(page).to have_content('Scholar@UC License Descriptions')
```
Comment (crowesn) [inline spec/features/end_to_end_spec.rb]: This text is unrelated to the license wizard and as-is the test should pass regardless of whether the button is showing. Need to fix this to look for the text from the button label, e.g. 'License Wizard'.

**Comment by lawhorkl** [comment]
should be good to go - fixed format and spec issues.

also, conflicted about help text from scholar 2. talked with @crowesn and we feel we should hold off for now and see how necessary it is, and possibly add in a later issue.

image

Other than that, this PR can be merged.

**Comment by crowesn** [comment]
Button doesn't load for me until refresh on local.

out

**Comment by lawhorkl** [comment]
@crowesn after applying you suggestion we're getting errors in travis that doesn't happen on local environments. Perhaps some issue with turbolinks.

**Comment by lawhorkl** [comment]
it looks like the license wizard button isn't being rendered when run through travis.

image

**Comment by lawhorkl** [comment]
add contextual help in license form (cant select as license)

change button color to white

**Comment by hortongn** [comment]
@lawhorkl Add these lines to the .travis.yml in your branch and then push it up to see if it gets all of your tests passing:
https://github.com/uclibs/scholar_uc/blob/e5058ac358e681c6d58646a2769059066253d256/.travis.yml#L14-L20

Here is a passing Travis build in a branch I based on your license branch:
https://travis-ci.org/uclibs/scholar_uc/builds/174623355

**Comment by lawhorkl** [comment]
@crowesn it is working fine on my end:

image

image

**Comment by crowesn** [comment]
This is still not working for me - can we get someone else from the team to fire it up and try?
@uclibs/developers 

out

**Comment by lawhorkl** [comment]
should be working NOW.
